redaxo/src/addons/debug/boot.php

```diff
@@ -8,6 +8,44 @@
 if (rex::isBackend() && 'debug' === rex_request::get('page')) {
     $index = file_get_contents(rex_addon::get('debug')->getAssetsPath('clockwork/index.html'));
 
+    $editor = rex::getProperty('editor');
+    $editorBasepath = rex::getProperty('editor_basepath');
+
+    $siteKey = rex::getServer().'redaxo/'.rex_url::backendController(['page' => 'structure'] + rex_api_debug::getUrlParams(), false);
```

Comment:
`rex::getServer().'redaxo/'`
bin damit net zufrieden und habe atm keine bessere möglichkeit gefunden an die absolute Backendurl zu kommen (http://example.org/redaxo/).

Reply:
Könnte das bei mehreren Domains zum Problem werden?

Reply:
Das ist der grund warum ich in der editor integration kein pfad mapping zulasse.. die integration ist nur aktiv wenn die lokalen pfade klar sind und man nicht von den backend urls mappen muss.

Reply:
> Das ist der grund warum ich in der editor integration kein pfad mapping zulasse.. die integration ist nur aktiv wenn die lokalen pfade klar sind und man nicht von den backend urls mappen muss.

man kann doch über `editor_basepath` diese mappen, oder meinst du was anders ?

bin am überlegen das sonst aus der aktuellen URL heraus zu parsen.

Hier hätten wir, wie generell mit den Settings "das Problem", dass man die settings net in die Browser Extension rein bekommen wird (denke ich zumindest).

Reply:
> man kann doch über editor_basepath diese mappen, oder meinst du was anders ?

Prinzipiell geht da viel... aktuell gehts aber nur mit lokalen pfaden in lokalen redaxo installationen

Reply:
> Hier hätten wir, wie generell mit den Settings "das Problem", dass man die settings net in die Browser Extension rein bekommen wird (denke ich zumindest).

Das sollte man als feedback im upstream issue anbringen

Reply:
Kannst du nicht statt mit `rex::getServer` direkt mit der `$_SERVER`-Variable, bzw. mit `rex_server(...)` arbeiten?